astropy/utils/data.py

```diff
@@ -1019,90 +1019,103 @@ def download_file(remote_url, cache=False, show_progress=True, timeout=None):
     else:
         url_key = remote_url
 
-    try:
-        if cache:
-            # We don't need to acquire the lock here, since we are only reading
-            with _open_shelve(urlmapfn, True) as url2hash:
-                if url_key in url2hash:
-                    return url2hash[url_key]
+    url_key_list = url_key.split(',')
```

Comment:
Just thinking aloud here: Should we make the delimiter an optional keyword (with comma as default) instead of hardcoding a comma? Is there any chance `remote_url` can have comma as part of its values?